moviepy/video/VideoClip.py

```diff
@@ -1337,6 +1337,19 @@ def search(string, arg):
 
 
 class BitmapClip(VideoClip):
+    DEFAULT_COLOR_DICT = {
```

Comment:
I have reviewed this pull in depth and I see everything correct. I don't ask you for a complete review because I understand that the test is a bit complex, but I would like to ask you if this `DEFAULT_COLOR_DICT` class property could have any downside. Also, do you agree with the name `DEFAULT_COLOR_DICT` @tburrows13?

Reply:
Yep, I'm not going to pretend to understand the test, but I really like the `DEFAULT_COLOR_DICT` change